pkg/config/data/config_schema_v1.0.json

```diff
@@ -85,6 +85,34 @@
               {
                 "$id": "#/properties/build/properties/run/items/anyOf/0",
                 "type": "string"
+              },
+              {
+                "$id": "#/properties/build/properties/run/items/anyOf/1",
+                "type": "object",
+                "properties": {
+                  "command": {
+                    "type": "string"
+                  },
+                  "mounts": {
+                    "type": "array",
+                    "items": {
+                      "type": "object",
+                      "properties": {
+                        "type": {
+                          "type": "string",
+                          "enum": ["secret"]
+                        },
+                        "id": {
+                          "type": "string"
+                        },
+                        "target": {
+                          "type": "string"
+                        }
+                      },
+                      "required": ["type", "id", "target"]
+                    }
+                  }
+                }
```

Comment:
Setting `"command"` as a required key causes `gojsonschema` to emit a validation error, despite the key being present (at least I'm pretty sure it is). I ran this through an online validator and couldn't reproduce this problem.